agenta-cli/agenta/cli/main.py

```diff
@@ -79,7 +79,9 @@ def cli():
 
 @click.command()
 @click.option("--app_name", default="")
-def init(app_name: str):
+@click.option("--backend_host", default="")
+@click.option("--init_option", default="")
```

Comment:
I think we should remove init_option, and skip asking for it in case the user has provided the app_name and backend_host (see suggestion below)